tests/cpgrid/zoltan_test.cpp

```diff
@@ -53,6 +52,35 @@ void MPI_err_handler(MPI_Comm *, int *err_code, ...){
 }
 #endif
 
+struct MPIFixture
+{
+    MPIFixture()
+    {
+#if HAVE_MPI
+    int m_argc = boost::unit_test::framework::master_test_suite().argc;
+    char** m_argv = boost::unit_test::framework::master_test_suite().argv;
+    helper = &Dune::MPIHelper::instance(m_argc, m_argv);
+#ifdef MPI_2
+    MPI_Comm_create_errhandler(MPI_err_handler, &handler);
+    MPI_Comm_set_errhandler(MPI_COMM_WORLD, handler);
+#else
+        MPI_Errhandler_create(MPI_err_handler, &handler);
+        MPI_Errhandler_set(MPI_COMM_WORLD, handler);
+#endif
+#endif
+    }
+    ~MPIFixture()
+    {
+#if HAVE_MPI
+        MPI_Finalize();
+#endif
+    }
+    Dune::MPIHelper* helper;
+    MPI_Errhandler handler;
```

Comment:
@blattms: This data member (`handler`) should **probably** be guarded by `#if HAVE_MPI`.  The post-merge sequential build [failed](https://ci.opm-project.org/job/opm-grid/81/console) with the diagnostic message
```
/var/lib/jenkins/post-builder/workspace/opm-grid/tests/cpgrid/zoltan_test.cpp:79:5: error: 'MPI_Errhandler' does not name a type; did you mean 'MPIError'?
     MPI_Errhandler handler;
     ^~~~~~~~~~~~~~
     MPIError
```

Reply:
Thanks, I could figure jenkins description of the problem